src/bench/block_assemble.cpp

```diff
@@ -73,9 +73,12 @@ static void AssembleBlock(benchmark::State& state)
     boost::thread_group thread_group;
     CScheduler scheduler;
     {
-        ::pblocktree.reset(new CBlockTreeDB(1 << 20, true));
-        ::pcoinsdbview.reset(new CCoinsViewDB(1 << 23, true));
-        ::pcoinsTip.reset(new CCoinsViewCache(pcoinsdbview.get()));
+        {
```

Comment:
style-nit: I created the above scope to prevent "leaking" of variable names further down, so no need to nest scopes here, just close/open the existing one.

Reply:
Fixed!